openapi-cli/src/main/java/org/ballerinalang/ballerina/ConverterUtils.java

```diff
@@ -0,0 +1,121 @@
+        Map<String, BLangExpression> attrMap = new HashMap<>();
+
+        for (RecordLiteralNode.RecordField field : list) {
+            if (field.isKeyValueField()) {
+                BLangRecordLiteral.BLangRecordKeyValueField attr = (BLangRecordLiteral.BLangRecordKeyValueField) field;
+                attrMap.put(attr.getKey().toString(), attr.getValue());
+            } else {
+                BLangRecordLiteral.BLangRecordVarNameField varNameField =
+                        (BLangRecordLiteral.BLangRecordVarNameField) field;
+                attrMap.put(varNameField.variableName.value, varNameField);
+            }
+        }
+
+        return attrMap;
+    }
+
+    /**
+     * Coverts the string value of an annotation attachment to a string.
+     *
+     * @param valueNode The annotation attachment.
+     * @return The string value.
+     */
+    public static String getStringLiteralValue(ParenthesizedArgList valueNode) {
+        SeparatedNodeList<FunctionArgumentNode> arg = valueNode.arguments();
+        return arg.get(0).toString();
+    }
+
+    /**
+     * Retrieves a specific annotation by name from a list of annotations.
+     *
+     * @param name        name of the required annotation
+     * @param pkg         package of the required annotation
+     * @param annotations list of annotations containing the required annotation
+     * @return returns annotation with the name <code>name</code> if found or
+     * null if annotation not found in the list
+     */
+    public static AnnotationAttachmentNode getAnnotationFromList(String name, String pkg,
+                                                                 List<? extends AnnotationAttachmentNode> annotations) {
+        AnnotationAttachmentNode annotation = null;
+        if (name == null || pkg == null) {
+            return null;
+        }
+
+        for (AnnotationAttachmentNode ann : annotations) {
+            if (pkg.equals(ann.getPackageAlias().getValue()) && name.equals(ann.getAnnotationName().getValue())) {
+                annotation = ann;
+            }
+        }
+
+        return annotation;
+    }
+
+    public static String convertBallerinaTypeToOpenAPIType(String type) {
+        String convertedType;
+        switch (type) {
+            case Constants.INT:
+                convertedType = "integer";
+                break;
+            case Constants.STRING:
+                convertedType = "string";
+                break;
+            case Constants.BOOLEAN:
+                convertedType = "boolean";
+                break;
+            case Constants.ARRAY:
+                convertedType = "array";
+                break;
+            case Constants.RECORD:
+                convertedType = "object";
+                break;
+            case Constants.DECIMAL:
+                convertedType = "decimal";
+                break;
+            default:
+                convertedType = "";
+        }
+        return convertedType;
+    }
```

Comment:
IMO keeping Enum definitions for in-built ballerina types and open api types would make more sense. WDYT?

Reply:
yeah it's better to keepnig  Enum definition, will change with Enum type definition